.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -0,0 +1,225 @@
+
+          export NOW_TEST=$(gomplate -i '{{ (time.Now).Format time.RFC3339}}');
+          poetry run rasa test nlu --no-plot -u dataset/$DATASET/test -m models/$DATASET/$CONFIG --out results/$DATASET/$CONFIG
+
+          echo "::set-output name=test_run_time::$(gomplate -i '{{ $t := time.Parse time.RFC3339 (getenv "NOW_TEST") }}{{ (time.Since $t).Round (time.Second 1) }}')"
+          echo "::set-output name=total_run_time::$(gomplate -i '{{ $t := time.Parse time.RFC3339 (getenv "NOW_TRAIN") }}{{ (time.Since $t).Round (time.Second 1) }}')"
+
+      - name: Download artifact
+        uses: actions/download-artifact@v2
+        continue-on-error: true
+        with:
+          name: report.json
+
+      - name: Generate a JSON file with a report
+        env:
+          SUMMARY_FILE: "./report.json"
+          DATASET: ${{ matrix.dataset }}
+          RESULT_DIR: "${{ github.workspace }}/results"
+          CONFIG: ${{ matrix.config }}
+          TEST_RUN_TIME: ${{ steps.run_test.outputs.test_run_time }}
+          TRAIN_RUN_TIME: ${{ steps.run_test.outputs.train_run_time }}
+          TOTAL_RUN_TIME: ${{ steps.run_test.outputs.total_run_time }}
+        run: |-
+          poetry run python .github/scripts/mr_generate_summary.py
+          cat $SUMMARY_FILE
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: report.json
+          path: ./report.json
+
+      - name: Publish results to Segment
+        env:
+          SEGMENT_TOKEN: ${{ secrets.SEGMENT_TOKEN }}
+          DATASET_NAME: ${{ matrix.dataset }}
+          CONFIG: ${{ matrix.config }}
+          TEST_RUN_TIME: ${{ steps.run_test.outputs.test_run_time }}
+          TRAIN_RUN_TIME: ${{ steps.run_test.outputs.train_run_time }}
+          TOTAL_RUN_TIME: ${{ steps.run_test.outputs.total_run_time }}
+          RESULT_DIR: "${{ github.workspace }}/results"
+        run: |-
+          export PR_URL="https://github.com/${GITHUB_REPOSITORY}/pull/${{ github.event.number }}"
+          poetry run pip install analytics-python
+          poetry run python .github/scripts/mr_publish_results.py
+
+
+      - name: Publish results to Segment
+        env:
+          SEGMENT_TOKEN: ${{ secrets.SEGMENT_TOKEN }}
+          DATASET_NAME: ${{ matrix.dataset }}
+          CONFIG: ${{ matrix.config }}
+          ACCELERATOR_TYPE: "GPU"
+          TEST_RUN_TIME: ${{ steps.run_test.outputs.test_run_time }}
+          TRAIN_RUN_TIME: ${{ steps.run_test.outputs.train_run_time }}
+          TOTAL_RUN_TIME: ${{ steps.run_test.outputs.total_run_time }}
+          RESULT_DIR: "${{ github.workspace }}/results"
+          PR_URL: ""
+        run: |-
+          poetry run pip install analytics-python
+          poetry run python .github/scripts/mr_publish_results.py
+
+  remove_runner_gpu:
+    name: Delete Github Runner - GPU
+    needs:
+    - deploy_runner_gpu
+    - model_regression_test_gpu
+    runs-on: ubuntu-latest
```

Comment:
do we need a `if: always()` since this should also run in case the testing failed?

Reply:
Right, thanks for 👀 !